aicsimageio/readers/reader.py

```diff
@@ -6,20 +6,65 @@
 
 import dask.array as da
 import numpy as np
+import xarray as xr
+from fsspec.spec import AbstractFileSystem
 
-from .. import types
-from ..dimensions import Dimensions
+from .. import transforms, types
+from ..dimensions import DEFAULT_DIMENSION_ORDER, DimensionNames, Dimensions
 from ..types import PhysicalPixelSizes
 
 ###############################################################################
 
 
 class Reader(ABC):
-    _dask_data = None
-    _data = None
+    _xarray_dask_data = None
+    _xarray_data = None
     _dims = None
     _metadata = None
 
+    @staticmethod
+    @abstractmethod
+    def _reader_supports_image(fs: AbstractFileSystem, path: str) -> bool:
```

Comment:
i would suggest removing the word reader from this api call since it is a method on a Reader class.

Reply:
hrmmmm. How about `image_supported`?

I am just think in terms of code syntax:

```python
if _reader_supports_image:
    ...

# vs

if _image_supported:
    ...
```

Reply:
seems fine, it is also a common convention to begin boolean-valued functions with `is_`